packages/suite-web/pages/_app.tsx

```diff
@@ -20,6 +21,8 @@ import { Store } from '@suite-types';
 import ImagesPreloader from '../support/ImagesPreloader';
 import { CypressExportStore } from '../support/CypressExportStore';
 
+const Tor = dynamic(() => import('@suite-support/Tor'), { ssr: false });
```

Comment:
why dynamic?

Reply:
NextJS doesn't like references to `window` when building the project. It's a stupid workaround but including it dynamically does the trick.

Reply:
so maybe `Tor` module should be wrapped into `<NoSSR />` component?

Reply:
Unfortunately that doesn't work. It's still trying to do some build time magic on it.

Reply:
ok, lets leave it like this, but i think it can be refactored to avoid this